src/core/rubberbandmodel.h

```diff
@@ -91,6 +92,7 @@ class QFIELD_CORE_EXPORT RubberbandModel : public QObject
 
     QgsPoint firstCoordinate() const;
     QgsPoint lastCoordinate() const;
+    QgsPoint penultimateCoordinate() const;
```

Comment:
Can we get some documentation for this property? That class has poor documentation coverage to begin with but we should make sure newly added stuff gets documented.

Reply:
Absolutely, on the list. I would suggest we also document things that are not public, for me it helps a lot to know what is expected to happen with `mStuff` properties.